examples/cra-kitchen-sink/src/stories/__snapshots__/addon-a11y.stories.storyshot

```diff
@@ -1,46 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Storyshots Addon a11y Default 1`] = `
-<button
-  disabled={false}
-  onClick={[Function]}
-  style={Object {}}
->
+<button>
   
 </button>
 `;
 
 exports[`Storyshots Addon a11y Disabled 1`] = `
 <button
-  disabled={true}
-  onClick={[Function]}
-  style={Object {}}
+  disabled=""
 >
   Testing the a11y addon
 </button>
 `;
 
 exports[`Storyshots Addon a11y Invalid contrast 1`] = `
 <button
-  disabled={false}
-  onClick={[Function]}
-  style={
-    Object {
-      "backgroundColor": "black",
-      "color": "black",
-    }
-  }
+  style="color:black;background-color:black"
 >
   Testing the a11y addon
 </button>
 `;
 
 exports[`Storyshots Addon a11y Label 1`] = `
-<button
-  disabled={false}
-  onClick={[Function]}
-  style={Object {}}
->
+<button>
```

Comment:
Do you have an idea why Enzyme strips attributes?

Reply:
its the renderer specified in the tests. you have `shallow` which shallowrenders, `mount` which more or less renders the entire vdom, and `render` (uses cheerio under the  hood) which just prints out html. see https://github.com/airbnb/enzyme/blob/master/docs/api/render.md

like i [said](https://github.com/storybooks/storybook/pull/2414#discussion_r154498667), we cant use `mount` with multisnapshot because the snapshots then include the `context` prop, and part of that is the file path. and the file path changes depending on what the absolute path on the system the snapshot is stored at

Reply:
im not very familiar with the internals of jest-specific-snapshots, but if there was a way for it to pass a relative path in the context, that would eliminate the problem. its where i see the solution to the problem at least.
i wouldnt want to modify the output of the renderer (like replacing the filepath with an empty string or somesuch), which is the only alternative i can think of